src/components/popup/transaction-list.tsx

```diff
@@ -1,22 +1,26 @@
-import { createTxDateFormatList } from '@common/group-txs-by-date';
+import { createTxDateFormatList } from '@common/transactions/transaction-utils';
+import { AddressTransactionWithTransfers } from '@stacks/stacks-blockchain-api-types';
 import { Box, Stack, Text, color } from '@stacks/ui';
-import React from 'react';
-import { TxItem } from './tx-item';
+import React, { useMemo } from 'react';
+import { TxWithTransferItem } from './tx-item';
 
-interface TransactionListProps {
-  txsByDate: ReturnType<typeof createTxDateFormatList>;
+interface TransactionWithTransferListProps {
+  txs: AddressTransactionWithTransfers[];
 }
-export function TransactionList({ txsByDate }: TransactionListProps) {
+
+export function TransactionWithTransferList({ txs }: TransactionWithTransferListProps) {
```

Comment:
RE naming here, the notion of "transactions with transfer" is really an API implementation detail the the wallet's presentation of a tx list doesn't need to be concerned with. Let's keep the name as it was?